docs/pages/1-generate-https.md

```diff
@@ -132,32 +130,6 @@ sudo apt-get update
 sudo apt-get install docker-ce
 ```
 
-You will first need to add yourself to the Docker group. This consists of two steps:
-
- - Adding yourself to the group
- - Restarting the instance
-
-To add yourself to the Docker group:
-
-```bash
-sudo usermod -aG docker $USER
-
-# restart docker
-sudo service docker restart
-```
-
-Then restart the instance. It will kick you off, and you will need to ssh in again.
-
-```bash
-sudo reboot
-```
-
```

Comment:
This section to add to Docker group and reboot is still needed.

Reply:
Yes, it definitely is. You can have sudo but once you are added to the docker daemon it needs to be restarted. Try it out - start a droplet having sudo (this was my case, for example), then try running docker. You will get a permission denied. if you add yourself to the group, you still get it. You would need to add then completely restart to get the right functionality.

Likely if you have seen otherwise, perhaps Digital Ocean takes the steps to add you to the docker group and then start the daemon in that state, so it only _seems_ like you "just need sudo."

Reply:
I get permission denied if I don't sudo, but I'm in the habit of sudo'ing everything. I start docker with sudo and I issue any commands with sudo after that. I haven't had any issues doing this with DigitalOcean or with my AWS Bitnami LAMP stack.

Reply:
We also don't want to assume that the user is necessarily root, or that it's even a droplet. If we have the step, worst case they are already able to use docker and they don't do it. If we don't have the step, the user might run into the error and not know what to do (and this would be poor documentation on our part for not anticipating this!)

Reply:
It's a better idea to add the users to the docker group (and technically they don't need sudo, altough with docker they could escalate to it) vs. letting them use it by giving sudo. Generally, you don't want to be using sudo with Docker. With the docker group It should work without it.